dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h

```diff
@@ -1,8 +1,8 @@
 /*
- * NOTE: File generated by lpc_cfg_utils.py
+ * NOTE: File generated by gen_soc_headers.py
  * from MIMXRT595SFAWC/signal_configuration.xml
  *
- * Copyright (c) 2022, NXP
+ * Copyright (c) 2024 NXP
```

Comment:
I know the script (unfortunately) doesn't handle this, but we need to make these copyrights distinct, like so:
```suggestion
 * Copyright 2022, 2024 NXP
```

Reply:
OK, Updated.